includes/functions.php

```diff
@@ -16,7 +16,8 @@
 //FUTURE define('ALLSKY_CONFIG',  'XX_ALLSKY_CONFIG_XX');
 define('RASPI_CONFIG',   'XX_RASPI_CONFIG_XX');
 
-if (ALLSKY_HOME == "XX_ALLSKY_HOME_XX") {
+i// Split the placeholder so it doesn't get replaced if the update script is run multiple times.
```

Comment:
Typo.. `i` is outside the comment

Reply:
It works on my Pi, but probably produced a web error.  Good catch.